run_cmd.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```

Comment:
What's the goal of this file?
If you need a working example of `exec` with the kubernetes library you can look at the functions responsible for IXI modules packing, upload and extraction.

Reply:
This file was a draft at an attempt not to use kubectl. I'll remove it.